### DISCUSSION TITLE
Can we use our existing postgres sql databse with supabase

### DISCUSSION THREAD

**Discussion by prohith14** [supabase/supabase#19096] [Feature Requests]
- We want to use supabase with our futter application for the real time data updates, how ever we want to use our own postgres sql database

**Comment by encima** [comment]
This can be done already, see [here](https://supabase.com/docs/guides/realtime/bring-your-own-database).

Alternatively, you can self-host the Supabase platform and connect it to your existing database

**Reply by prohith14** [reply]
We are unable to do this 
"Alternatively, you can self-host the Supabase platform and connect it to your existing database" documentation is unclear
can you please help us here
while self hosting we tried using our database connection instead of creating new postgres db but we are getting errors

**Reply by prohith14** [reply]
level=fatal msg="running db migrations: Migrator: problem creating schema migrations: CREATE TABLE \"schema_migrations\" (\n\"version\" VARCHAR (14) NOT NULL\n);\nCREATE UNIQUE INDEX \"schema_migrations_version_idx\" ON \"schema_migrations\" (version);: ERROR: no schema has been selected to create in (SQLSTATE 3F000)"
